packages/gatsby-theme-api-docs/src/hooks/use-api-resources.js

```diff
@@ -3,7 +3,7 @@ import { useStaticQuery, graphql } from 'gatsby';
 export default () => {
   const queryResult = useStaticQuery(
     graphql`
-      fragment postOrPutMethod on RamlResourcePost {
+      fragment methods on RamlResourceMethod {
```

Comment:
Some methods do not have the `body`, we should differentiate the fragments between the different methods. We can have a "base" method fragment, then for the methods that have the extra fields 

```js
fragment methodBody on ??? {
  applicationjson {
    type
    builtinType
  }
}
fragment methodFields on RamlResourceMethod {
  # all fields, without body
}

query GetAllRamlResources {
  allRamlResource {
    nodes {
      post {
        ...methodFields
        body {
          ...methodBody
        }
      }
    }
  }
}
```